docs/accessibility-testing.md

```diff
@@ -0,0 +1,37 @@
+# Accessibility testing
+
+This build has a very high standard for accessibility testing. The merge request template and accessibility checklist in the [Monday ticket template](https://torchbox.monday.com/boards/1192293412/pulses/1349479497) contain prompts for what must be tested.
+
+## Automated checks
+
+Automated checks can be tested with the [Axe browser plugin](https://www.deque.com/axe/devtools/chrome-browser-extension). There should be zero issues reported, both at desktop and mobile.
```

Comment:
Suggestion: Add a link to the extension for Firefox - https://www.deque.com/axe/devtools/firefox-browser-extension/

Reply:
done! thanks chris.